functions.sh.in

```diff
@@ -88,7 +88,8 @@ readconfig() {
 	else
 		if [ -n "$gitbase" ]; then
 			case $(git config remote.origin.url) in
```

Comment:
```suggestion
			case $(git remote get-url origin) in
```
The value from config is not necessarily a URL, but gets translated to one by `url.*` configuration.